tst/testbugfix/00069.tst

```diff
@@ -0,0 +1,3 @@
+# 2005/05/05 (TB)
+gap> IsBound( ShowPackageVariables );
+true
```

Comment:
Looks like a relic from times when we were supplying bugfix archives, to unpack and overwrite existing files. Not sure we need this any more.